include/zephyr/bluetooth/gatt.h

```diff
@@ -666,10 +676,14 @@ ssize_t bt_gatt_attr_read_chrc(struct bt_conn *conn,
  *  attribute value.
  *
  *  @param _uuid Characteristic attribute uuid.
- *  @param _props Characteristic attribute properties.
- *  @param _perm Characteristic Attribute access permissions.
- *  @param _read Characteristic Attribute read callback.
- *  @param _write Characteristic Attribute write callback.
+ *  @param _props Characteristic attribute properties,
+ *		  a bitmap of @c BT_GATT_CHRC_* macros.
+ *  @param _perm Characteristic Attribute access permissions,
+ *		 a bitmap of @ref bt_gatt_perm values.
```

Comment:
```suggestion
 *  @param _props Characteristic attribute properties,
 *                a bitmap of @c BT_GATT_CHRC_* macros.
 *  @param _perm Characteristic Attribute access permissions,
 *               a bitmap of @ref bt_gatt_perm values.
```

Please use spaces instead of tabs when indenting after a non-whitespace character

Reply:
Sure. Fixed all of these, I think.

However, what's the actual full rule about tabs after non-whitespace characters? Because there's plenty of those in that very file beside my changes: aligning macro definitions, field names in structs and newline-escaping backslashes. Is this only about doxygen comments?

Out of all things Zephyr took from the Linux kernel, I wish tabs weren't one of them :joy: